spec/pluralrules.html

```diff
@@ -70,7 +68,7 @@
            1. Let _fv_ be the substring of _s_ from position _dp_, exclusive.
            1. Let _f_ be ? ToNumber(_fv_).
            1. Let _v_ be ? ToLength(? Get(_fv_, *"length"*)).
-         1. Let _i_ be ? ToNumber(_iv_).
+         1. Let _i_ be ? abs(? ToNumber(_iv_)).
```

Comment:
Line 67: `Let _iv_ be the substring of _s_ from position 0, inclusive, to position _dp_, exclusive.`